pkg/services/tree/replicator.go

```diff
@@ -36,6 +42,12 @@ func (s *Service) replicationWorker() {
 		select {
 		case <-s.closeCh:
 			return
+		case op := <-s.replicateLocalCh:
```

Comment:
... and local ones (supposed to be fast) here? What's gonna happen if we're waiting for the network operation to finish (5s * number of endpoints)?

Reply:
Yeah, actually it is even worse: we could deadlock here under a very high load. Fixed.